x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { apm, timerange } from '@elastic/apm-synthtrace';
+import datemath from '@elastic/datemath';
+import expect from '@kbn/expect';
+import { range } from 'lodash';
+import { AlertType } from '../../../../plugins/apm/common/alert_types';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { createAndRunApmMlJob } from '../../common/utils/create_and_run_apm_ml_job';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const synthtraceEsClient = getService('synthtraceEsClient');
+  const ml = getService('ml');
+  const supertest = getService('supertest');
+
+  registry.when(
+    'fetching service anomalies with a trial license',
+    { config: 'rules', archives: ['apm_mappings_only_8.0.0'] },
+    () => {
+      const spikeStart = datemath.parse('now-2h')!.valueOf();
+      const spikeEnd = datemath.parse('now')!.valueOf();
+
+      const start = datemath.parse('now-2w')!.valueOf();
+      const end = datemath.parse('now')!.valueOf();
```

Comment:
I think we should avoid dates relatives to now if possible. It will make the tests more stable, e.g. when dealing with DST changes.

Reply:
`windowSize` is configurable no? Although that is also a relative date. Let's leave it then, probably not worth it.